setup.py

```diff
@@ -41,7 +41,8 @@
         'termcolor == 1.*',
         'natsort == 5.0.*',
         'appdirs == 1.4.*',
-        'python-dateutil == 2.7.*'
+        'python-dateutil == 2.7.*',
+        'PyInquirer == 1.0.*'
```

Comment:
```suggestion
        'PyInquirer == 1.0.*',
```

let's start putting trailing commas in here, so we don't have extraneous diffs in the future?